### DISCUSSION TITLE
Labs mLOGO does not work with 4k 24 or 4K 30 on my GoPro 11

### DISCUSSION THREAD

**Discussion by F-Trindade** [gopro/labs#522] [Q&A]
Hi guys.

It is the exact oposit of "works on my computer"...

My Logo works on 1080p but NOT in 4k, any frame rate.
I had it in 25p... didn't work.
Now I changed Gopro to 30fps.. still doesn't. (but1080p is still ok)

Any ideas?
Thanks

This is  similar to another topic but in my case it is JUST above 1080p that it doesn't work~. (in 2.7K 60 also works)

**Comment by F-Trindade** [comment]
It's the 10bit.... Logo in 4k/30  only works if 10bit is off :( 
Not good.

**Reply by dnewman-gpsw** [reply]
Correct, LOGO hack was created for HERO8, using GP1 and was 8-bit only.  There is no 10-bit support for overlays.

**Reply by F-Trindade** [reply]
Yeah... 
But that is really bad. I mean, why will I shoot in 8bit if I have 10bit? My logo only makes sense in  final production and I would only add it here if it means not having to edit the footage on a PC...

This is useless for me. 

But wait...
I think it works in 10bit in 1080 and 2.7k... I didn't change that option while testing and it worked... only in 4k it didn't...
Hm....

**Reply by dnewman-gpsw** [reply]
1080 and 2.7k modes are 8-bit, which is why it works.

**Reply by F-Trindade** [reply]
Hahahah you're right!!! I never used those modes, I didn't even know that!!! Thanks.
